src/switch_snmp.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python
+#
+# igcollect - Switch port traffic over SNMP
+#
+# Copyright (c) 2016, InnoGames GmbH
+#
+
+from __future__ import print_function
+from argparse import ArgumentParser
+
+import re
+import subprocess
+import time
+
+port_name_oid = '.1.3.6.1.2.1.31.1.1.1.1'
+procurve_trunk_oid = '.1.3.6.1.4.1.11.2.14.11.5.1.7.1.3.1.1.8'
+cisco_agg_oid = '.1.2.840.10006.300.43.1.2.1.1.12'
+
+COUNTERS = {
```

Comment:
It doesn't look like something needs to be a `dict`.  I would go for a tuple of tuples.  Tuple is also more suitable to globals, because of being immutable.